client/me/help/main.jsx

```diff
@@ -211,14 +218,57 @@ class Help extends React.PureComponent {
 				onClick={ this.trackCoursesButtonClick }
 				href="https://wordpress.com/webinars"
 				target="_blank"
-				title={ this.props.translate( 'Courses' ) }
-				description={ this.props.translate(
+				title={ translate( 'Courses' ) }
+				description={ translate(
 					'Learn how to make the most of your site with these courses and webinars'
 				) }
 			/>
 		);
 	};
 
+	supportSessionCard = () => {
+		const { translate } = this.props;
+
+		return (
+			<Card className="help__support-session-card">
+				<div className="help__support-session-text">
+					<h2 className="help__support-session-title">
+						{ translate( 'Schedule a support session' ) }
+					</h2>
+					<p className="help__support-session-description">
+						{ translate(
+							'Quick Start Support Sessions give you a way to talk to one of our Happiness Engineers via a screen share with audio.'
+						) }
+					</p>
+					<div className="help__support-session-actions">
+						<Button
+							className="help__support-session-action"
+							primary
+							href={ localizeUrl( 'https://wordpress.com/checkout/offer-quickstart-session/' ) }
+							onClick={ this.trackSupportSessionButtonClick() }
```

Comment:
If we're deciding not to do anything we tracking, we can remove this entirely.

Unless we just want to keep this placeholder here. It does nothing though. If we want to keep this here, let's from the `()`. Instead of removing it `onClick`, this is invoking the functionality immediately as it is rendered.